gebru/roster.json

```diff
@@ -82,6 +82,10 @@
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "video",
+          "url": "https://youtu.be/jVLV4fjyDls"
```

Comment:
@AverKill 
I know the name of the key is misleading, but can you please replace the full url with the video_id
i.e, "url": "jVLV4fjyDls"